packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -83,12 +92,14 @@ export interface MultiSelectProps<T> extends ListItemsProps<T>, SelectPopoverPro
      * N.B. the behavior of this prop differs slightly from the same one
      * in the Suggest component; see https://github.com/palantir/blueprint/issues/4152.
      *
+     * This prop is ignored if `customTarget` is supplied.
+     *
      * @default false
      */
     openOnKeyDown?: boolean;
 
     /**
-     * Input placeholder text. Shorthand for `tagInputProps.placeholder`.
+     * Placeholder text. Used as placeholder for TagInput or Input depending on whether `customTarget` is supplied.
```

Comment:
I don't think we need to commit to this much implementation detail in the docs
```suggestion
     * Placeholder text used for the filter input element.
```